### PR TITLE
Adding four new faculty for University of Wuerzburg

### PR DESCRIPTION
**Pull request by IngoScholtes**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR PULL REQUEST WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [x] All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

- [x] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [x] Combine multiple updates to a single institution into a **single PR.**

- [x] Only submit one pull request per institution.

- [x] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [x] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [x] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [x] Check to make sure that you have no spaces after commas, or any missing fields.

- [x] Check to make sure the home page is correct.

- [x] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [x] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [x] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

### CODE REVIEW

File: csrankings-c.csv [hunk 1]
```diff
@@ -192,6 +192,7 @@ Carola Wenk,Tulane University,http://www.cs.tulane.edu/~carola,JLWWzugAAAAJ
 Carole A. Goble,University of Manchester,http://www.manchester.ac.uk/research/Carole.goble,2G4kqPYAAAAJ
 Carole Anne Goble,University of Manchester,http://www.manchester.ac.uk/research/Carole.goble,2G4kqPYAAAAJ
 Carole Twining,University of Manchester,http://personalpages.manchester.ac.uk/staff/carole.j.twining,23t6LBIAAAAJ
+Carolin Wienrich,University of Würzburg,https://www.mcm.uni-wuerzburg.de/piis/people/prof-dr-carolin-wienrich/,3O8K2pIAAAAJ
```
Comment (emeryberger) [inline csrankings-c.csv:195]: Not in CS.

File: csrankings-t.csv [hunk 2]
```diff
@@ -382,6 +382,7 @@ Thomas Chatain,Ecole Normale Superieure de Cachan,http://www.lsv.ens-cachan.fr/~
 Thomas Courtade,Univ. of California - Berkeley,https://people.eecs.berkeley.edu/~courtade,xRmmtzIAAAAJ
 Thomas D. LaToza,George Mason University,https://cs.gmu.edu/~tlatoza,200KNLAAAAAJ
 Thomas D. Nielsen,Aalborg University,http://people.cs.aau.dk/~tdn,6fWF0CgAAAAJ
+Thomas Dandekar,University of Würzburg,https://www.biozentrum.uni-wuerzburg.de/bioinfo/research/groups/funct-genomics-systems-biology/people/thomas-dandekar/,bQR_Kc8AAAAJ
```
Comment (emeryberger) [inline csrankings-t.csv:385]: Not in CS.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by emeryberger** [comment]
> Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.

**Comment by IngoScholtes** [comment]
Thank you for the question. Both are tenured and have an appointment at the Faculty of Math and Computer Science. They can thus independently supervise PhD theses in CS. I will add links to a page showing the courtesy appointment.

**Pull request merged**